src/test/java/domain/CardRankTest.java

```diff
@@ -0,0 +1,54 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class CardRankTest {
+
+    public static Stream<Arguments> provideJQK() {
+        return Stream.of(Arguments.of(CardRank.JACK, CardRank.QUEEN, CardRank.KING));
+    }
+
+    @DisplayName("카드의 숫자는 1부터 k까지 14개다.")
+    @Test
+    void cardNumber() {
+        //given
+
+        //when
+        CardRank[] values = CardRank.values();
+
+        //then
+        assertThat(values).hasSize(14);
+    }
+
+    @DisplayName("카드의 숫자는 1부터 10까지의 점수를 가진다.")
+    @Test
+    void cardNumberPoint() {
+        //given
+        CardRank five = CardRank.FIVE;
+
+        //when
+
+        //then
+        assertThat(five.getPoint()).isEqualTo(5);
```

Comment:
given, when, then 주석을 적으셨으니, 가능하면 given, when, then 구조를 지켜서 테스트 코드를 짜보시면 좋을 것 같아요. 😃 
테스트 코드도 유지보수 대상이라는 측면에서 이 구조를 지키며 테스트 코드 짜는 것이 매우 의미있다고 생각해요.  😉 

```suggestion
        //when
        int actual = five.getPoint()
        
        //then
        assertThat(actual).isEqualTo(5);
```

Reply:
좋은 피드백 감사합니다! 
given, when, then 구조가 단순한 템플릿이 아닌 유지보수 측면에서 더 의미 있는 방식이라는 점을 알 수 있었습니다.
피드백 반영하여 개선해보겠습니다!